src/components/Tag.js

```diff
@@ -73,6 +74,24 @@ const Title = styled(TextElement).attrs(({ color, theme: { colors } }) => ({
   marginBottom: 1,
 });
 
+const openUrlWithWarningAlert = url => {
+  Alert({
+    buttons: [
+      {
+        onPress: () => Linking.openURL(url),
+        text: 'Open',
+      },
+      { style: 'cancel', text: 'Cancel' },
+    ],
+    message:
+      'In a moment you will be redirected to\n' +
+      url +
+      '\n' +
+      'Make sure this address is safe, if in doubt do not open it.',
+    title: 'Warning',
+  });
+};
+
```

Comment:
no, we don't do this in Tokens, when the website is provided so we should not do this here. I suggest filtering out `http://`

Reply:
I'll be filtering http then